hack/lib/protoc.sh

```diff
@@ -76,13 +93,32 @@ function kube::protoc::protoc() {
   )
 }
 
+# Generates $1/api.pb.go from the protobuf file $1/api.proto without using gogo
+# $1: Full path to the directory where the api.proto file is
+function kube::protoc::protoc() {
+  local package=${1}
+
+  (
+    cd "${package}"
+
+    protoc \
+      --go_out=. \
+      --go_opt=paths=source_relative \
+      --go-grpc_out=. \
+      --go-grpc_opt=paths=source_relative \
+      api.proto
+  )
+}
+
 # Formats $1/api.pb.go, adds the boilerplate comments and run gofmt on it
 # $1: Full path to the directory where the api.proto file is
 function kube::protoc::format() {
   local package=${1}
 
-  # Update boilerplate for the generated file.
-  cat hack/boilerplate/boilerplate.generatego.txt "${package}/api.pb.go" > tmpfile && mv tmpfile "${package}/api.pb.go"
+  # Update boilerplate for the generated files.
+  for file in "${package}"/api*.pb.go ; do
```

Comment:
if this makes more filenames now, this bit in `codegen::protobindings()` needs updating:
```
    for api in "${apis[@]}"; do
        git_find -z ":(glob)${api}"/'**/api.pb.go' \
            | xargs -0 rm -f
    done
```

Reply:
True, fixed that